ci-operator/step-registry/openshift-qe/ocp-61589-max-nodes-network-expansion/openshift-qe-ocp-61589-max-nodes-network-expansion-commands.sh

```
@@ -0,0 +1,321 @@
+    
+    local start_time
+    start_time=$(date +%s)
+    local timeout_seconds
+    timeout_seconds=$(echo "$timeout" | sed 's/h/*3600/g; s/m/*60/g; s/s//g' | bc)
+    
+    # Create node provisioning progress log
+    echo "timestamp,elapsed_seconds,total_nodes,ready_nodes,notready_nodes,pending_nodes" > "$RESULTS_DIR/node-provisioning-progress.csv"
+    
+    while true; do
+        local ready_count
+        ready_count=$(oc get nodes --no-headers | grep " Ready " | wc -l)
```

Shellcheck (0.11.0)

[style] Consider using 'grep -c' instead of 'grep|wc -l'.

(SC2126)